### DISCUSSION TITLE
Blazor com .NET 8 na prática - Paginação

### DISCUSSION THREAD

**Discussion by Otavio-Wollny** [balta-io/forum#434] [Tira Dúvidas] [answered]
Boa noite.

Você tem algum tutorial sobre paginação no blazor consumindo dados do BD?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Otavio-Wollny, como vai?
Segue:
https://balta.io/cursos/fundamentos-entity-framework/skip-take-e-paginacao-de-dados
